import-areas/import-areas.go

```diff
@@ -353,7 +354,7 @@ func main() {
 
 	logData = log.Data{"AreaData": walesData}
 
-	if err = session.DB("areas").C("areas").Insert(walesData); err != nil {
+	if _, err = session.DB("areas").C("areas").Upsert(bson.M{"id": "W92000004"}, walesData); err != nil {
```

Comment:
Are these ID's constant (or are the likely to change)? Better off having these in a `map[string]string` like:

`var regionalIDs = map[string]string{
	"England": "E92000001",
	"Wales": "W92000004"
}`

And referencing that map rather than inlining.